### DISCUSSION TITLE
Solved: IE00B42NKQ00

### DISCUSSION THREAD

**Discussion by Heidieker** [ghostfolio/ghostfolio#1499] [Q&A]
Hi,
i have an etf IE00B42NKQ00, that is recognized wrong. How may i correct this?

My IE00B42NKQ00 ist this -> https://www.justetf.com/de/etf-profile.html?isin=IE00B42NKQ00

Ghostfolio is recognizing this-> https://finance.yahoo.com/quote/IESU.L/

How can i put the correct ETF in my Portfolio?

**Comment by dtslvr** [comment]
Hi @Heidieker

What about [QDVF.DE](https://finance.yahoo.com/quote/QDVF.DE)?

I searched for "iShares S&P 500 Energy Sector" directly at _Yahoo Finance_.

**Reply by Heidieker** [reply]
Thank you! I tried this on justetf, not on yahoo.
